requirements.txt

```diff
@@ -1,2 +1,4 @@
+neuro-flow
 tqdm==4.33.0
 catalyst==19.12
+wandb==0.8.2
```

Comment:
let's bump it to the latest available `wandb==0.10.14` and fixate `neuro-flow==21.1.4`

Reply:
Apparently, it was more than just version bump, but it's done now :)